syntax = "proto3";

package tensorflow;

message TokenProto {
  int64 token_id = 1;
  string token_str = 2;
};

message TokenModelProto {
  int64 token_sos = 1;
  int64 token_eos = 2;
  int64 token_eow = 3;
  int64 token_unk = 4;
  int64 token_blank = 5;

  repeated TokenProto tokens = 6;
};
